src/AssocValue.php

```diff
@@ -143,6 +144,18 @@ public function only(...$keys): AssocValue;
      */
     public function withoutElement($value): AssocValue;
 
+    /**
+     * @phpstan-return AssocValue<int|string, TKey>
+     */
+    public function flip(): AssocValue;
+
+    /**
+     * @param TKey $keyA
+     * @param TKey $keyB
+     * @phpstan-return AssocArray<TKey, TValue>
+     */
+    public function swap($keyA, $keyB): AssocArray;
```

Comment:
It can have type-hint `int|string` for arguments, and return should be `AssocValue`